apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -375,6 +375,29 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
             if (unitType) return unitType
         }).filter(Boolean)
     }, [initialHasAllProperties, initialNewsItemScopes, initialProperties.length, initialSectionKeys])
+    const commonTemplates = useMemo(() => {
+        const commonTemplates = {}
+
+        for (const id in templates) {
+            if (templates[id].type === NEWS_TYPE_COMMON || isNull(templates[id].type)) {
+                commonTemplates[id] = templates[id]
+            }
+        }
+
+        return commonTemplates
+    }, [templates])
+
+    const emergencyTemplates = useMemo(() => {
+        const commonTemplates = {}
+
+        for (const id in templates) {
+            if (templates[id].type === NEWS_TYPE_EMERGENCY || isNull(templates[id].type)) {
```

Comment:
Is it ok, that templates without `type` will be presented in `commonTemplates` and in `emergencyTemplates`?